README.md

```diff
@@ -9,3 +9,22 @@ See https://github.com/nation3/agreements-app
 ## Documentation
 
 https://github.com/nation3/agreements-app/blob/main/README.md#documentation
+
+## How to deploy/redeploy subgraph
+Update `networks.json` file before deploying subgraph
```

Comment:
Update how?  Using `yarn install`?

Reply:
![image](https://github.com/nation3/agreements-contracts/assets/42999269/38e4f71b-9c39-4a06-8ab3-5c5033943cd7)
when you deploy smartcontract to the new address, or you want to track more smart contracts

Reply:
Okay, I understand now 😄